lib/schema/resolvers/Mutation/createInvitation.ts

```diff
@@ -37,7 +37,9 @@ export const createInvitation: NonNullable<
   // Doesn't work without it :/
   await sql`
       UPDATE public.worker AS u
-      SET workerusername = ${input.email_address}
+      SET
+          workerusername = ${input.email_address},
+          workermodifieddate = NOW()
```

Comment:
as a nice to have can also include a `workermodifiedby` set to some worker instance id in cust 0